src/overlays/actors/ovl_En_Bombf/z_en_bombf.h

```diff
@@ -6,9 +6,20 @@
 
 struct EnBombf;
 
+typedef void (*EnBombfActionFunc)(struct EnBombf*, struct GlobalContext*);
+
 typedef struct EnBombf {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xC4];
+    /* 0x014C */ ColliderCylinder bombCollider;
+    /* 0x0198 */ ColliderJntSph jntSphList;
+    /* 0x01B8 */ ColliderJntSphItem explosionCollider[1];
```

Comment:
There is something wrong with these names.
The collider should probably be named `explosionCollider` while the item list is named `explosionColliderItems`.

Reply:
My understanding of JntSph was that the ColliderJntSph was just the list containing all of the items, while the item itself is the actual collider that contains the relevant information like dim and body

Reply:
Well the items contain data on their individual information, sure, but that doesn't change the fact that the "collider" is the main thing, not its items.

Reply:
aight makes sense, will change